### PR TITLE
(nomacs) fixed updater and msi installer

### PR DESCRIPTION
**Pull request by majkinetor**
#901

### CODE REVIEW

**Comment by majkinetor** [comment]
Can't see how this conflict happened. Otherwise this is done

File: automatic/nomacs/nomacs.nuspec [hunk 1]
```diff
@@ -3,63 +3,58 @@
...
-  - rotate images
-  - drag and drop images
-  - automatically save screenshots
-  - delete/rename images
-  - create mosaic image
-  - set wallpaper (windows only)
+- crop images
+- resize images
+- pseudo color function
+- color adjustments (brightness, contrast, saturation â€¦)
```
Comment (AdmiringWorm) [inline automatic/nomacs/nomacs.nuspec]: Seems like you have somehow gotten an invalid character here somehow. Would need to be fixed

**Comment by RedBaron2** [comment]
I kind of remember there being now x86 & x64 versions for this software. Also the developer listing `/passive` as the unattended switch on the FAQ page.

**Comment by AdmiringWorm** [comment]
@RedBaron2 if you look through the scripts, you'll notice both x86 and x64 is being aquired for the installer (the portable one only supports x64).

The installer is a normal msi, those support both the `/passive` and the `/quiet` switches.

**Comment by RedBaron2** [comment]
OK mobile view of site was limiting viewing. Carry on.

**Pull request merged**